src/templates/Practice5.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+
+
+interface Data {
```

Comment:
Could you change this name into something more specific like `Todo` or `Task`?

`Data` doesn't say much about what's inside 🧐

Reply:
Made the changes